### PR TITLE
cleanup the readme

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +131,5 @@ curl http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar > /tmp/php-cs-fixer.
 php /tmp/php-cs-fixer.phar  fix -v --diff --using-cache=yes src/
 ```
 
-TODO: 
-- Examples
+TODO:
 - ResolverInterface still needed?
```
Comment (dbu) [inline README.md]: can we decide on this one? for the sake of customizing, i think we should keep it.

File: README.md [hunk 2]
```diff
@@ -88,7 +131,5 @@ curl http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar > /tmp/php-cs-fixer.
 php /tmp/php-cs-fixer.phar  fix -v --diff --using-cache=yes src/
```
Comment (dbu) [inline README.md:135]: this should be in the ci instead...

Comment (chregu) [reply README.md:135]: Not with `fix`, just the test, right? But there's no CI tests, currently

File: README.md (hunk 1 above)
Comment (chregu) [reply README.md]: Too long ago, since I used it. If it helps customizing, sure, let's keep it.

File: README.md (hunk 2 above)
Comment (dbu) [reply README.md:135]: oh right, needs to be test then. created #4

File: README.md (hunk 1 above)
Comment (dbu) [reply README.md]: i removed the todo

**Comment by dbu** [comment]
can you double-check if i got the descriptions of the operations in the readme right, or messed up something? some of it like seoLanguage was guessing ;-)

File: README.md [hunk 3]
```diff
@@ -1,17 +1,20 @@
 # rokka/twig
 
-A twig extension to easily use [rokka](https://rokka.io) related methods in your twig templates.
+A twig extension to use [rokka](https://rokka.io) related methods in your twig templates.
 
-Docs are coming, very much work in progress, see the source or open an issue, if questions.
-
-Stable release coming soon.
+This allows you to upload images to rokka while rendering a template. As this will slow down rendering the template
+considerably, this is mainly useful with static site generators that render the pre-render the templates before
```
Comment (chregu) [inline README.md]: Only on the first hit, now? Once an image is uploaded and we have stored the hash somewhere (by default in that .json) it shouldn't significantly slow down the rendering (apart from reading that json file)

Comment (dbu) [reply README.md]: i rephrased to be clearer. also made clear that the twig functions also work with rokka hashes, so the extension as such definitely makes sense in all scenarios.

**Pull request merged**